tests/DependencyInjection/RabbitMqConsumerHandlerExtensionTest.php

```diff
@@ -135,33 +135,50 @@ public function testConfigureContainerParameter(
 		$this->compile();
 	}
 
-	public function testConfigureCustomLoggerInstance(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function configureContainerServiceDataProvider(): Generator
```

Comment:
Here we can also incorporate the default aliases.

Reply:
Incorporated

Reply:
Now duplicate as well, same as parameters.

Reply:
Should be deduplicated properly now.